src/Data/Quaternion/Rotation.purs

```diff
@@ -107,3 +112,28 @@ act (Rotation p) v =
   unsafePartial $ case Vec.toArray v of
     [x, y, z] ->
       vectorPart (Quaternion zero x y z `conjugateBy` p)
+
+-- | Though all functions in the library that create a Rotation ensure that
+-- | the underlying Quaternion has unit magnitude, some computations may result
+-- | in the magnitude drifting away from 1.0 for numerical or other reasons.
```

Comment:
Maybe not the best choice of words. By "numerical" I meant accumulation of rounding error. As opposed to the linear interpolation case where the interpolated quaternion is expected to have a significantly different magnitude.